### DISCUSSION TITLE
Brave browser very slow

### DISCUSSION THREAD

**Discussion by volitionn** [celzero/rethink-app#1609] [Q&A] [answered]
Hi, I'm using RethinkDNS with a wireguard config file from Mullvad and everything works fine except that I get very slow speeds when using Brave browser specifically. When I use the Mullvad client app it works fine but when RethinkDNS is on it gets very slow. I tried tweaking various settings in brave://flags but nothing worked. Is there any way I can troubleshoot the issue ? Thanks

**Comment by ignoramous** [comment]
Hm, strange.

What app version are you using? Check the footer of the "About" UI.

If you're technical enough, you can start a *Packet capture* from *Configure* -> *Settings* and load it up in any tool that will help analyse a PCAP file (like [WireShark](https://archive.is/FGIhS)).

**Comment by volitionn** [comment, marked as answer]
It's working fine now. Looks like the problem was on Brave's end and got fixed in their latest update.

**Reply by ignoramous** [reply]
Thanks for circling back! Appreciate it.
